addons/hr_contract/models/resource.py

```diff
@@ -1,15 +1,17 @@
-# -*- coding:utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+from collections import defaultdict
 from datetime import datetime
 
-from odoo import fields, models
+from odoo import _, api, fields, models
 from odoo.osv.expression import AND
 
 
 class ResourceCalendar(models.Model):
     _inherit = 'resource.calendar'
 
     contracts_count = fields.Integer("# Contracts using it", compute='_compute_contracts_count', groups="hr_contract.group_hr_contract_manager")
+    running_contracts_count = fields.Integer("Running contracts count", compute='_compute_contracts_count', groups="hr_contract.group_hr_contract_manager", search="_search_running_contracts_count")
+    contract_ids = fields.One2many('hr.contract', 'resource_calendar_id')
```

Comment:
```suggestion
    contract_ids = fields.One2many('hr.contract', 'resource_calendar_id', groups="hr_contract.group_hr_contract_manager"
)
```
for me this field should also be restrict to employee with contract manager